src/App/Pages/Accounts/LoginSsoPageViewModel.cs

```diff
@@ -207,5 +226,19 @@ private async Task LogIn(string code, string codeVerifier, string orgId)
                     AppResources.AnErrorHasOccurred);
             }
         }
+
+        private async Task<bool> TryClaimedDomainLogin()
+        {
+            var userEmail = await _stateService.GetRememberedEmailAsync();
+            var claimedDomainOrg = await _organizationService.GetClaimedOrganizationDomainAsync(userEmail);
+            if (!string.IsNullOrEmpty(claimedDomainOrg))
+            {
+                OrgIdentifier = claimedDomainOrg;
+                await LogInAsync();
+                return true;
+            }
+
+            return false;
```

Comment:
Usually this is done the other way around, so you check and `return false` first:
```c#
if (string.IsNullOrEmpty(claimedDomainOrg))
{
    return false;
}

OrgIdentifier = claimedDomainOrg;
await LogInAsync();
return true;
```

Reply:
You are right.